testsuite/openmodelica/cppruntime/fmu/modelExchange/2.0/testDrumBoiler.mos

```diff
@@ -157,148 +157,84 @@ val(evaporator_qm_S, 100);
 //   </ScalarVariable>
 //   <!-- Index of variable = \"7\" -->
 //   <ScalarVariable
-//     name=\"der(evaporator.T_D)\"
```

Comment:
These variables shouldn't be suppressed.

Reply:
removed filtering of dummy states and dummy derivative vars